addons/api/addon/models/role.js

```diff
@@ -103,6 +113,43 @@ export default class RoleModel extends GeneratedRoleModel {
     return resolve([]);
   }
 
+  /**
+   * A promise that resolves to an array of Group instances.
+   * When calling this getter, be sure to await resolution
+   * before interacting with the results.
+   * @type {Promise[GroupModel]}
+   */
+  get managedGroups() {
```

Comment:
If we are aiming for an async function (based on the coment above) shouldn't we define this function as `async`?